eng/Versions.props

```diff
@@ -50,7 +50,8 @@
   <PropertyGroup>
     <MicrosoftDotNetApiCompatVersion>9.0.0-beta.23621.2</MicrosoftDotNetApiCompatVersion>
     <MicrosoftDotNetCodeAnalysisPackageVersion>9.0.0-beta.23621.2</MicrosoftDotNetCodeAnalysisPackageVersion>
-  </PropertyGroup>
+    <MicrosoftDotNetGenAPIVersion>9.0.0-beta.23621.2</MicrosoftDotNetGenAPIVersion>
```

Comment:
Please include a subscription for this version in versiondetails.xml so that it's not stuck at this version and gets updated automatically.

Reply:
Also, we'd prefer if you use `Microsoft.DotNet.GenAPI.Task` instead.  The version from arcade is [no longer being maintained](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.GenAPI/README.md) and is missing support for the latest language features.